Makefile

```diff
@@ -149,7 +148,6 @@ generate-mocks:
 	mockery --name '.*' --dir=fvm --case=underscore --output="./fvm/mock" --outpkg="mock"
 	mockery --name '.*' --dir=fvm/state --case=underscore --output="./fvm/mock/state" --outpkg="mock"
 	mockery --name '.*' --dir=ledger --case=underscore --output="./ledger/mock" --outpkg="mock"
-	mockery --name 'SubscriptionManager' --dir=network/ --case=underscore --output="./network/mocknetwork" --outpkg="mocknetwork"
```

Comment:
where is the 2nd generation of this mock? I'm looking for another instance of `mockery --name 'SubscriptionManager'`

Reply:
`mockery` generates all the mocks for `./network` down there:

```
mockery --name '.*' --dir=network --case=underscore --output="./network/mocknetwork" --outpkg="mocknetwork"
```

Reply:
Thanks for the clarification.